### ISSUE TITLE
task for copying only jars 

### ISSUE THREAD

**Issue by amiracam** [jruby-gradle/jruby-gradle-plugin#310]
I have used the following:

task copyRuntimeLibs(type: Copy) {
    into "lib"
    from configurations.jrubyJar
}

which grabs both jars and gems. hoping for a way of specifying what file type I want or at least rejecting gem files but  my gradle is weak

**Comment by ysb33r** [comment]
Use exclude ANT patterns. Something like this.

```groovy
task copyRuntimeLibs(type: Copy) {
  into "lib"
  from configurations.jrubyJar, {
    exclude '**/*.gem'
  }
}
```

Alternatively use an ANT include pattern (you can combine both excludes and includes).

```groovy
task copyRuntimeLibs(type: Copy) {
  into "lib"
  from configurations.jrubyJar, {
    include '**/*.jar'  
  }
}
```

**Comment by amiracam** [comment]
yes, thanks, I should have worked harder for this before bothering, I did this:

exclude "*.gem"
